Code/GraphMol/SmilesParse/catch_tests.cpp

```diff
@@ -254,6 +254,34 @@ TEST_CASE("github #2257: writing cxsmiles", "[smiles,cxsmiles]") {
     auto smi = MolToCXSmiles(*mol);
     CHECK(smi == "CN |$_AV:val1;val2$,atomProp:0.p2.v2:1.p1.v1|");
   }
+  SECTION("enhanced stereo 1") {
+    auto mol = "C[C@H](F)[C@H](C)[C@@H](C)Br |a:1,o1:4,5|"_smiles;
+    REQUIRE(mol);
+    auto smi = MolToCXSmiles(*mol);
+    CHECK(smi == "C[C@H](F)[C@H](C)[C@@H](C)Br |a:1,o1:4,5|");
```

Comment:
Is this sufficient to check the cx smiles round tripping or should we check the mol.getStereoGroups?

I note that is a StereoGroups == operator ..

Reply:
We have tests to make sure that the stereogroup info is parsed correctly:
https://github.com/rdkit/rdkit/blob/master/Code/GraphMol/SmilesParse/cxsmiles_test.cpp#L417